website/forms/features/debug.md

```diff
@@ -6,3 +6,36 @@ title: Debugging
 The debugging screen provides configuration options and overrides default functionalities for developers, but caution should be used as these options can break form functionality.
 
 ![Debug screen](/img/forms/debug.webp)
+
+
+## Bypass validation
+
+When a form is submitted, an internal validator is triggered to validate the form. Once all internal validations are successfully passed, the form data is sent to the integration. With this toggle, you have the option to bypass internal validation for all forms and directly send the data to the integration.
+
+## Bypass captcha
+
+This toggle allows sending the form without CAPTCHA validation while enabling the feature. It is helpful for testing or running automated tests.
+
+## Don’t clear form after submission
+
+After the form is successfully submitted, all fields will be reset to their original state. This toggle will allow duplicate submissions with the same data.
+
+## Developer mode
+
+Enabling the developer mode will provide various small yet helpful features to simplify the debugging process.
```

Comment:
Maybe provide a few examples of features here?

Reply:
The list of these features is on the plugin, so I would like to avoid duplication :D I will add a comment